modules/core/shared/src/main/scala/lucuma/core/math/dimensional/quantity.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.dimensional
+
+import coulomb._
+import monocle.Focus
+import monocle.Lens
+
+/**
+ * A magnitude of type `N` and a runtime representation of a physical unit.
+ */
+trait Qty[N] {
```

Comment:
Isn't this essentially the same as `Quantity`? Why do we need it?

Reply:
`Quantity[N, U]` is a value class wrapping an `N`. The unit `U` is erased at runtime. Also, if we want to define a value that can have different units we are in trouble since `U` is part of the type definition of a `Quantity`.

`Qty` hides the units from the type definition and moves it to a runtime value.

Better name suggestions are very welcome, please!